content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Upgrade to InfluxDB OSS 2.0rc0
+description: >
+  Upgrade to InfluxDB OSS 2.0rc0.
+menu:
+  influxdb_2_0_ref:
+    name: Upgrade to InfluxDB OSS 2.0rc0
+weight: 9
+---
+
+To upgrade from InfluxDB 2.0 beta 16 or earlier to InfluxDB 2.0rc0, you must manually upgrade all resources and data to the latest version by completing the following steps:
+
+- [1. Disable existing integrations](#)
+- [2. Stop existing InfluxDB beta instance](#)
+- [3. (Optional) Rename existing InfluxDB binaries](#)
+- [4. Move existing data and start the latest InfluxDB](#)
+- [5. Start old InfluxDB beta instance](#)
+- [6. Create configuration profiles for the InfluxDB CLI](#)
+- [7. Copy all resources from old instance to the new one](#)
+- [8. Setup integrations to point to new instance](#)
+- [9. Load historical data into new instance](#)
+
+Depending on how you have things set up and how important the data stored in InfluxDB is,
+you can choose the parts of this guide that make sense to you.
```

Comment:
This wording is a little strange to me. Maybe "...you can determine which steps apply to you."